### ISSUE TITLE
Mouse offset with Dual Screen Setup on OSX

### ISSUE THREAD

**Issue by fhibler** [dwservice/agent#8]
If you remote control an OSX desktop with more than one display connected to it, there is an offset which does not seem to be predictable. Might also be due to the different resolutions on the two displays. Anyone else ran into it yet?

I am happy to demonstrate and replicate if needed. 

Thanks for the good work!

**Comment by dwservice** [comment]
Yes other users have same problem, but they don't unknown why. Thanks your info I can replicate the issue.
